viz_scripts/plots.py

```diff
@@ -181,16 +181,16 @@ def overeall_energy_impact(x,y,color,data,plot_title,file_name):
     
     
     
-def energy_impact(x,y,color,plot_title,file_name):
+def energy_impact(x,y,color,plot_title,file_name,xl='Energy_Impact(kWH)'):
     color = color.map({True: 'green', False: 'red'})
-    objects = ('Energy Savings', 'Energy Loss')
+    objects = ('Savings', 'Loss')
```

Comment:
Clever option to avoid passing in two parameters!

Reply:
Leidy thought these labels weren't very intuitive for the graphs of cost and time!